js/main.js

```diff
@@ -1,28 +1,118 @@
+$('document').ready(function(){
+
+
 /*
 Apple
 The apple tree (Malus pumila, commonly and erroneously called Malus domestica) is a deciduous tree in the rose family best known for its sweet, pomaceous fruit, the apple. It is cultivated worldwide as a fruit tree, and is the most widely grown species in the genus Malus. The tree originated in Central Asia, where its wild ancestor, Malus sieversii, is still found today. Apples have been grown for thousands of years in Asia and Europe, and were brought to North America by European colonists. Apples have religious and mythological significance in many cultures, including Norse, Greek and European Christian traditions.
+*/
+
 
+$('.apple').click(function() {
```

Comment:
Watch your indentation here. These should be indented 1 tab since they are nested inside of the `$(document).ready(function() { ... });`